frontend/test/metabase/scenarios/admin/settings/public-sharing.cy.spec.js

```diff
@@ -0,0 +1,236 @@
+import { restore, modal, enableActionsForDB } from "__support__/e2e/helpers";
+import { SAMPLE_DB_ID } from "__support__/e2e/cypress_data";
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { ORDERS_ID } = SAMPLE_DATABASE;
+
+describe("scenarios > admin > settings > public sharing", () => {
```

Comment:
We have never had tests for public sharing settings. So it took quite some time for me to set this up.